src/app/backend/resource/daemonset/list.go

```diff
@@ -117,9 +117,10 @@ func toDaemonSetList(daemonSets []apps.DaemonSet, pods []v1.Pod, events []v1.Eve
 	daemonSetList.ListMeta = api.ListMeta{TotalItems: filteredTotal}
 
 	for _, daemonSet := range daemonSets {
+		desired := daemonSet.Status.DesiredNumberScheduled
```

Comment:
Looking throught the golang refspec I don't seen anything that the temp local variable in the scope of this for block is not allowed to be reused. So in a way, I believe a smarter go compiler might break us again. How about also capturing the index in the loop and passing `&daemonSets[i].Status.DesiredNumberScheduled`

Reply:
I like that route better as well, thanks for the input. 

I'll try to churn that out today and verify it works. :)

Reply:
Updated and verified. Thanks @rf232!